protocol/messenger.go

```diff
@@ -991,6 +993,12 @@ func (m *Messenger) Init() error {
 		}
 
 		m.allChats.Store(chat.ID, chat)
+
+		if i < 60 {
```

Comment:
Magic number? What is it?

Reply:
Magic number

Reply:
this function is a temporary workaround, the entire implementation is not perfect, I could add a description but...it won't explain anything :)

Reply:
in will only add more questions :)